front/tests/pages/operational-studies/operational-studies-page.ts

```diff
@@ -265,5 +282,34 @@ class OperationalStudiesPage extends CommonPage {
   async checkNumberOfTrains(number: number) {
     await expect(this.trainTimetable).toHaveCount(number);
   }
+
+  async checkInputsBeforeEditingAPacedTrain(
+    translations: ManageTrainScheduleTranslations,
+    editedPacedTrainTimeWindow: string,
+    editedPacedTrainInterval: string
+  ) {
+    await expect(this.definePacedTrainCheckbox).toBeChecked();
+    await expect(this.pacedTrainTimeWindow).toHaveValue(
+      String(Duration.parse(editedPacedTrainTimeWindow).total('minute'))
+    );
+    await expect(this.pacedTrainIntervalInput).toHaveValue(
+      String(Duration.parse(editedPacedTrainInterval).total('minute'))
+    );
```

Comment:
You can extract this into a separate function and reuse it here as well as in `checkPacedTrainModeAndVerifyInputs()`
```suggestion
    await expect(this.pacedTrainTimeWindow).toBeVisible();
    await expect(this.pacedTrainTimeWindow).toHaveValue(DEFAULT_PACED_TRAIN_SETTINGS.timeWindow);

    await expect(this.pacedTrainIntervalInput).toBeVisible();
    await expect(this.pacedTrainIntervalInput).toHaveValue(DEFAULT_PACED_TRAIN_SETTINGS.interval);
```

Reply:
done